streamz/core.py

```diff
@@ -1063,7 +1070,7 @@ def cb(self):
         while True:
             L, self._buffer = self._buffer, []
             metadata, self.metadata_buffer = self.metadata_buffer, []
-            m = [m for l in metadata for m in l]
+            m = [m for ml in metadata for m in ml]
```

Comment:
This was part of the checkpointing PR. This time around (because of flake8 updates), it was showing flake8 [errors](https://github.com/scipy/scipy/issues/8389) saying that the name of variable `l` is ambiguous. That's why I changed the name to `ml` instead.